### DISCUSSION TITLE
Bluetooth slot...

### DISCUSSION THREAD

**Discussion by Agio56** [NightscoutFoundation/xDrip#3115] [Q&A]
I'm trying to connect the Anubis G6 transmitter with two phones: using Xdrip on one and BYOD on the other. I set Xdrip to slot 1.
Xdrip connects normally but BYOD on the other phone never connects.
My newbie question is: is it possible to check which slot Xdrip is using?
In System Status the Transmitter Days item shows the value 0/0.5: is this correct?
Thanks for the support

**Comment by Navid200** [comment]
If you disable collection in xDrip, does Dexcom app connect then?

https://navid200.github.io/xDrip/docs/Stop-xDrip.html

**Reply by Agio56** [reply]
Hi Navid, thank you for your constant support.
I followed your advice: if I disable XDRip BYOD does not connect to Anubis.
I tried with two different devices, no connection with Anubis G6.
Can I try to reset transmitter?

**Reply by Navid200** [reply]
There is no reason.
If xDrip works and Dexcom app does not, the only explanation is that you have not disabled xDrip.

Changing the transmitter slot number is not a safe practice.  It is only meant for testing unless we figure out a way to make it reliable.  That's why it is behind engineering mode.
As is, it is not recommended to do.

**Reply by Agio56** [reply]
I can't explain why the Dexcom BYOD won't connect to the Anubis g6 transmitter. I disabled Xdrip and turned off the phone, then attempted several times to connect BYOD to the transmitter with another phone, without success.
Anyway I followed your advice and disabled engineering mode in Xdrip

**Reply by Navid200** [reply]
If you hard reset the transmitter, does Dexcom app still not connect?

**Reply by Agio56** [reply]
Dear friend, I don't know why and how but I managed to connect the Anubis transmitter (without hard reset) to the Dexcom app. Not only that, but now I can successfully use slot 1 with Xdrip and slot 2 with Dexcom!!
Thanks to your support I am learning a lot and quickly (not bad for my age!!)
